cmd/initialize/initlizeMonorepo.go

```diff
@@ -41,27 +41,55 @@ func runInitMonorepo(cmd *cobra.Command, args []string) {
 	initproject.IsMonorepo = true
 	initproject.RunInitProject(cmd, args)
 
-	// Get apps to initialize
-	appsOfMonorepoDir, err := prompt.PromptForMonorepoApps(cmd)
-	if err != nil {
-		printer.Error(cmd, err)
-		return
-	}
-
 	m, err := manifest.Restore()
 	if err != nil {
 		printer.Error(cmd, err)
 		return
 	}
 
-	// Initialize each app
-	for _, appDir := range appsOfMonorepoDir {
-		printer.Info(fmt.Sprintf("Initializing app %s", appDir))
-		err := initializeMonorepoApp(cmd, appDir, orgID, m.Config, appService, envService, m.Secret)
+	for {
+		appPath, err := prompt.PromptString(cmd, "Provide the path to an application:")
+		if err != nil {
+			printer.Error(cmd, err)
+			return
+		}
+
+		cleanPath := filepath.Clean(appPath)
+
+		if strings.HasPrefix(cleanPath, "..") || strings.Contains(cleanPath, "../") {
```

Comment:
you should use https://pkg.go.dev/path/filepath#Rel and base the logic of the result of that.  This would simplify a lot of this logic :)

Reply:
Thanks, I will add this fix in #169. I fixed some issues there. I'm waiting for the reviews on the other ones, and I will do a cascade merge.

Reply:
Agreed with Matthew - use built in libraries when you can. Also, why can't we support relative paths that go up and over?